Assembler.Core/IMessageEnricher.cs

```diff
@@ -0,0 +1,11 @@
+﻿using Assembler.Core.Entities;
+
+namespace Assembler.Core
+{
+    public interface IMessageEnricher<TFrame, TMessage>
+    where TFrame : BaseFrame
+    where TMessage : BaseMessageInAssembly
```

Comment:
Indent the 2 lines above

Reply:
f

Reply:
👍

Reply:
You can resolve the conversation

Reply:
I don't have the resolve button :|
go figure